lib/components/user/nav-login-button.js

```diff
@@ -35,6 +39,10 @@ export default class NavLoginButton extends Component {
     profile: null
   }
 
+  _handleLinkClick = memoize(
+    item => () => this.props.routeTo(item.url)
+  )
+
```

Comment:
@landonreed @evansiroky please agree on the approach.

Reply:
Pretty much everywhere in our code we have made a habit of not inlining handlers. And typically this has not been via memoizing functions. I really prefer that we just stick to this habit for now. I still say we just create a very simple component (can be functional) that just takes an item as a prop and handles the click.

Reply:
@landonreed This will probably not work if I have to add a `<Nav>` element inside the mobile view: https://github.com/opentripplanner/otp-react-redux/blob/70a9d96b31722b836d3ec099609d100c245a1439/lib/components/mobile/navigation-bar.js#L58

Reply:
I found a better way by adding an onClick attribute to each link object.